docker-build.sh

```diff
@@ -26,7 +26,7 @@ prepare_extra_common() {
         ;;
     esac
     ./autogen.sh
-    ./configure --prefix=${TARGET_DIR} ${CROSS_OPT}
+    ./configure --prefix=${TARGET_DIR} --enable-lto ${CROSS_OPT}
```

Comment:
This is only set for libzimg.
Shouldn't it be set here for ffmpeg? https://github.com/jellyfin/jellyfin-ffmpeg/blob/32815a2ad139198b21716682a343ef4d630e17f8/debian/rules#L94-L95